specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-10-01/storagecache.json

```
@@ -1404,12 +1442,12 @@
           "type": "string"
         },
         "smbServerName": {
-          "description": "The name (NetBIOS) used for the HPC Cache to join the Active Directory Domain. Length must not be greater than 15 and chars must be in list of [-0-9a-zA-Z_] char class.",
-          "pattern": "^[-0-9a-zA-Z_]{1,15}$",
+          "description": "The name (NetBIOS) used for the HPC Cache to join the Active Directory domain. Length must not be greater than 15 and chars must be from the [-0-9a-zA-Z_] char class.",
```

Missed the underscore in the description's char class:
```suggestion
          "description": "The name (NetBIOS) used for the HPC Cache to join the Active Directory domain. Length must not be greater than 15 and chars must be from the [-0-9a-zA-Z] char class.",
```

---

Fixed in next commit.